composer.json

```diff
@@ -1,35 +1,48 @@
 {
-    "name": "treehouselabs/base-api-bundle",
-    "type": "symfony-bundle",
-    "description": "Base bundle for Api functionality",
-    "keywords": ["api"],
-    "license": "MIT",
-    "authors": [
-      {
-        "name": "Peter Kruithof",
-        "email": "peter@treehouse.nl"
-      },
-      {
-        "name": "Jeroen Fiege",
-        "email": "jeroen@treehouse.nl"
-      }
-    ],
-    "require": {
-      "php":                           ">=5.4",
-      "symfony/symfony":               "~2.3",
-      "doctrine/orm":                  "~2.3",
-      "jms/serializer":                "~0.14",
-      "friendsofsymfony/user-bundle":  "~2.0@dev",
-      "treehouselabs/keystone-bundle": "~1.0"
+  "name": "treehouselabs/base-api-bundle",
+  "type": "symfony-bundle",
+  "description": "Base bundle for Api functionality",
+  "keywords": [
+    "api"
+  ],
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Peter Kruithof",
+      "email": "peter@treehouse.nl"
     },
-    "require-dev": {
-      "phpunit/phpunit":               "~4.1",
-      "behat/behat":                   "~3.0",
-      "behat/symfony2-extension":      "~2.0@dev"
-    },
-    "autoload": {
-      "psr-4": {
-        "TreeHouse\\BaseApiBundle\\": "src/TreeHouse/BaseApiBundle/"
-      }
+    {
+      "name": "Jeroen Fiege",
+      "email": "jeroen@treehouse.nl"
+    }
+  ],
+  "require": {
+    "php": ">=5.5",
+    "symfony/symfony": "^2.6",
+    "jms/serializer": "^1.0"
+  },
+  "require-dev": {
+    "doctrine/orm": "^2.3",
+    "friendsofsymfony/user-bundle": "~2.0@dev",
+    "phpunit/phpunit": "^4.1",
+    "behat/behat": "^3.0",
+    "behat/symfony2-extension": "^2.0"
+  },
+  "suggest": {
+    "treehouselabs/keystone-bundle": "For API authentication that is easy to set up",
+    "behat/behat": "To test the API using Behat",
```

Comment:
Yeah I thought a bit about this: they are not required for using the bundle (hence the suggests), except when you use the Behat contexts. But if we want to test the contexts we would need them as dev dependencies.

It seemed more appropriate than putting them all in `require`: which would create dependencies to eg. Behat and FOSUserBundle, even if you're not using the Behat tests.